lib/util/naming.js

```diff
@@ -0,0 +1,110 @@
+/**
+ * @fileoverview Common helpers naming of plugins and formatters
```

Comment:
Nitpick: Seems to be missing a word here, maybe something like "Common helpers for naming of plugins and formatters"?